src/components/Image.js

```diff
@@ -1,32 +1,18 @@
 import React from 'react';
-import { useStaticQuery, graphql } from 'gatsby';
-import Img from 'gatsby-image';
+import PropTypes from 'prop-types';
 
-/*
- * This component is built using `gatsby-image` to automatically serve optimized
- * images with lazy loading and reduced file sizes. The image is loaded using a
- * `useStaticQuery`, which allows us to load the image from directly within this
- * component, rather than having to pass the image data down from pages.
- *
- * For more information, see the docs:
- * - `gatsby-image`: https://gatsby.dev/gatsby-image
- * - `useStaticQuery`: https://www.gatsbyjs.org/docs/use-static-query/
- */
+const Image = ({ alt, ...props }) => {
+  return <img alt={alt} {...props} />;
+};
 
-const Image = () => {
-  const data = useStaticQuery(graphql`
-    query {
-      placeholderImage: file(relativePath: { eq: "gatsby-astronaut.png" }) {
-        childImageSharp {
-          fluid(maxWidth: 300) {
-            ...GatsbyImageSharpFluid
-          }
-        }
-      }
-    }
-  `);
+Image.propTypes = {
+  src: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+  width: PropTypes.number,
+};
 
-  return <Img fluid={data.placeholderImage.childImageSharp.fluid} />;
+Image.defaultProps = {
+  width: 1200,
```

Comment:
Would it make more sense to make this width `auto` by default? Seems weird we'd want to specify a hard width for all images rather than let theme be their natural size until someone configures it otherwise. Thoughts?

Reply:
So my understanding of the ticket was that we wanted to standardize the size of the images (?) @mmfred @djsauble what are your thoughts on this?

Reply:
Great point 😄 . I'll be curious to see what the others say .

Reply:
Images that are too large in the vertical dimension make it hard to follow the flow of the page. For example:

<img width="1904" alt="Screen Shot 2020-07-13 at 10 52 14 AM" src="https://user-images.githubusercontent.com/186715/87337116-c9cf6580-c4f7-11ea-8541-68f41b1f6f7d.png">

In contrast, images that take the full width of the page can be fine as long as they're not too tall. For example:

<img width="1904" alt="Screen Shot 2020-07-13 at 10 56 14 AM" src="https://user-images.githubusercontent.com/186715/87337162-dfdd2600-c4f7-11ea-90fb-7f3f9822c6b9.png">

My suggest is to 1) set a max-height of 400px for images, 2) let the width be up to 100% while respecting the aspect ratio of the image, 3) left align images so they follow the shape of the text.

Reply:
Okay! thank you for those suggestions @djsauble